AnalyzeDotNetProject.csproj

```diff
@@ -1,12 +1,14 @@
 <Project Sdk="Microsoft.NET.Sdk">
-
+  
```

Comment:
Please delete trailing whitespaces. Review the [contributing guidelines](www.contributing..com.ar) before sending a PR.